app/assets/stylesheets/comments.scss

```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Comments controller here.
```

Comment:
I would remove all of these stylesheets that were generated from using generators as they are not needed a this point